Tests/Fixtures/Cache/Collections.php

```diff
@@ -9,18 +9,23 @@
 use MongoDB\BSON\ObjectId;
 
 /** @ODM\Document */
```

Comment:
Since the TestKernel configuration changed to `attribute`, is it necessary to keep legacy `annotations`?

Reply:
Nope, I was wondering if we should still test `annotations` when using Symfony 5.4 and then remove them when deprecating our `doctrine_mongodb.odm.metadata.annotation*` services

Reply:
I've added another commit removing the annotations 👍